vshard/router/init.lua

```diff
@@ -278,6 +288,8 @@ local function discovery_f(router)
         -- Step 2: map stage - send parallel requests for every
         -- iterator, prune orphan iterators whose replicasets were
         -- removed.
+        discovery_service:set_activity(
+            'sending requests for bucket discovery')
```

Comment:
Nit: there are no other requests in this service, don't need to say it is for "bucket discovery".

Reply:
Fixed